src/topology/homeomorph.lean

```diff
@@ -51,6 +51,17 @@ rfl
 @[continuity]
 protected lemma continuous (h : α ≃ₜ β) : continuous h := h.continuous_to_fun
 
+/-- Change the homeomorphism `f` to make the inverse function definitionally equal to `g`. -/
+def change_inv (f : α ≃ₜ β) (g : β → α) (hg : function.right_inverse g f) : α ≃ₜ β :=
```

Comment:
Should there also be a variant in which the given function `g` is a left inverse?

Reply:
We could, but I think it's fine to wait with that until we actually use it at some point.